src/testing/builder/CategoriesMapModelBuilder.js

```diff
@@ -0,0 +1,108 @@
+// @flow
+
+import { CategoriesMapModel, CategoryModel } from '@integreat-app/integreat-api-client'
+import moment from 'moment-timezone'
+import type { FileCacheStateType, LanguageResourceCacheStateType } from '../../modules/app/StateType'
+import seedrandom from 'seedrandom'
+import hashUrl from '../../modules/endpoint/hashUrl'
+
+const DEFAULT_MAX_WIDTH = 3
+const DEFAULT_DEPTH = 2
+const MAX_PREDICTABLE_VALUE = 6
+
+class CategoriesMapModelBuilder {
+  _depth: number
+  _maxWidth: number
```

Comment:
it's the `arity` of the tree. You can add a comment that the builder generates a perfect tree of categories.